pages/api/auction/getUserBids.ts

```diff
@@ -0,0 +1,17 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import { getUserBids } from '../../../server/utils/firebase-admin'
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (req.method !== 'GET') return res.status(400).json({ message: 'Only get is allowed' })
+  const { user } = req.body
```

Comment:
body won't be available for GET request, right? may be change to params

Reply:
Change implemented